interfaces/apparmor/backend.go

```diff
@@ -256,6 +264,27 @@ func addContent(securityTag string, snapInfo *snap.Info, opts interfaces.Confine
 	}
 }
 
+// procSelfMountInfo is exposed as a variable for testing anyEncryptedDirectory.
+var procSelfMountInfo = mount.ProcSelfMountInfo
+
+// anyEncryptedDirectory returns true ecryptfs is mounted anywhere.
+// TODO: Make this look for the place where the snap is mounted from, once that
+// information becomes available.
```

Comment:
Right, it's all there but we don't have it in snapd, it's thrown away the moment the task is complete. I will work wit @chipaca to fix this so that we can reliably detect the right condition.